packages/jitar-reflection/.npmignore

```diff
@@ -0,0 +1,4 @@
+src
+node_modules
+tsconfig.json
+*.tgz
```

Comment:
There are some entries missing in this config.

coverage
test
.eslintignore
.eslintrc
jest.config.js